apps/researcher/src/lib/community.test.ts

```diff
@@ -1,6 +1,40 @@
 import {describe, expect, it} from '@jest/globals';
 import {auth} from '@clerk/nextjs';
-import {isAdmin, Membership, sort, SortBy} from './community';
+import {isAdmin, sort, SortBy} from './community';
+import {OrganizationMembership} from '@clerk/nextjs/dist/types/server';
+
+const basicOrganization = {
+  id: 'organization1',
+  name: 'Organization 1',
+  slug: 'organization-1',
+  imageUrl: 'https://example.com/image.png',
+  createdAt: 1690000000000,
```

Comment:
Is there a way to use Date instead (makes the API friendlier)?

Reply:
Only if we build a wrapper around the clerk response. This comes directly from Clerk.